ui/imports/utils/LocaleUtils.qml

```diff
@@ -0,0 +1,22 @@
+pragma Singleton
+
+import QtQml 2.14
+
+QtObject {
+
+    function fractionalPartLength(num) {
+        if (Number.isInteger(num))
+            return 0
+
+        return num.toString().split('.')[1].length
```

Comment:
Just wonder if separator is always '.'

Reply:
when using toString than yes, it's not localized